code/global/Utils.cs

```diff
@@ -86,6 +86,21 @@ public static IEnumerable<Client> GetClientsSpectatingPlayer(TTTPlayer player)
             return clients;
         }
 
+        public static List<TTTPlayer> GetAlivePlayersByRoleName(string role)
```

Comment:
Not a bad idea either. I'd still probably want to compare the lowered names though.

Reply:
We should - if not already done - set the role's name to lowercase by default (to avoid interaction issues with outside resources or client input). Every role name has to be unique, so it doesn't matter. Should help avoiding issues in language interaction as well.

Reply:
Resolved with latest commit. Force role to auto .ToLower() and compared with the role instead